examples/lock-app/infineon/psoc6/README.md

```diff
@@ -139,53 +141,64 @@ commands. These power cycle the BlueTooth hardware and disable BR/EDR mode.
 
 ## Building with OPTIGA™ Trust M as HSM
 
-Infineon Hardware Security Module-OPTIGA™ Trust M is a high-end security solution that provides an anchor of trust for connecting IoT devices to the cloud, giving every IoT device its own unique identity. 
+Infineon Hardware Security Module-OPTIGA™ Trust M is a high-end security
+solution that provides an anchor of trust for connecting IoT devices to the
+cloud, giving every IoT device its own unique identity.
 
-OPTIGA™ Trust M offers a wide range of security features, making it ideal for industrial and building automation applications, smart homes and connected consumer devices.
+OPTIGA™ Trust M offers a wide range of security features, making it ideal for
+industrial and building automation applications, smart homes and connected
+consumer devices.
```

Comment:
This marketing stuff does not belong in the SDK. Please keep it to facts/technical. Remove this paragraph.

Reply:
Hi @tcarmelveilleux, Thanks for your review. we deleted this part in the latest commit.